lib/extruder.js

```diff
@@ -35,11 +35,15 @@ const extruder = {
         const res = {};
 
         if (options.frontMesh) {
-            res.frontMesh = extruder.getHorizontalGeom(horizontalPathsByDepth,innerPathsByDepth, [0], 0,true);
+            let invertFrontNormal = true;
+            if (options.doNotInvertFrontNormal) invertFrontNormal = false;
```

Comment:
why not directly assign like this :
```js
const inverseFrontNormal = !options.doNotInvertFrontNormal;
```

Reply:
Even more simplified by !options.doNotInvertFrontNormal